### PR TITLE
DOCS-2680-Callback-URL

### PR DESCRIPTION
**Pull request by hannanelson**
Added section in the mule-agent.yml config topic to point to existing section elsewhere. Minor cleanup.

### CODE REVIEW

File: modules/ROOT/pages/runtime-manager-agent-notifications.adoc [hunk 1]
```diff
@@ -9,23 +9,34 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid
...
++
 [source,yaml,linenums]
 ----
+transports:
   rest.agent.transport:
     enabled: true
     callback:
-      url: http://localhost:8080/
-    port: 8888
+      url: http://0.0.0.0:8081/test/
```
Comment (marialuzshin) [inline modules/ROOT/pages/runtime-manager-agent-notifications.adoc]: Is there a reason for changing this example? I'm not sure is that configuration will work.

Comment (hannanelson) [reply modules/ROOT/pages/runtime-manager-agent-notifications.adoc]: I used the example from this KB:
https://help.mulesoft.com/s/article/How-to-configure-a-callback-URL-to-forward-agent-notifications-in-mule-runtime

Comment (mcocaro) [reply modules/ROOT/pages/runtime-manager-agent-notifications.adoc]: this is incorrect - the correct value was the previous one.

**Review by mcocaro — CHANGES_REQUESTED** [review]
see comments

File: modules/ROOT/pages/runtime-manager-agent-notifications.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/runtime-manager-agent-notifications.adoc]: Reverted.

**Review by marialuzshin — APPROVED** [review]
LGTM!

**Review by mcocaro — APPROVED** [review]

File: modules/ROOT/pages/runtime-manager-agent-notifications.adoc [hunk 2]
```diff
@@ -9,23 +9,34 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid
...
+The Anypoint Runtime Manager agent publishes notifications, in JSON format, about events that occur in the Mule instance.
+This enables you to implement a system for receiving and handling notifications.
+Notifications are sent over both the REST and WebSocket transports.
+For more information about transports, see xref:runtime-manager-agent-architecture.adoc[Runtime Manager Agent Architecture].
 
-== Configure the Mule Agent Notifications Callback URL
+== Configure the Agent Notifications Callback URL
 
-To receive notifications on the specified URL, you must first configure the `callback` URL in the `mule-agent.yml` file inside the REST transport section as shown below.
+To forward Runtime Manager agent notifications to a URL over the REST transport, configure a `callback` URL in the mule-agent.yml file.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/runtime-manager-agent-notifications.adoc]: In line 131 of the previous file, you used tick marks: `mule-agent.yml`.

**Review by Mulesoft-Mysti — APPROVED** [review]
Looks great!

**Pull request merged**